pkg/operator/targetconfigcontroller/targetconfigcontroller.go

```diff
@@ -839,8 +848,14 @@ func ManageCSRIntermediateCABundle(ctx context.Context, lister corev1listers.Sec
 	csrSignerCA, err := client.ConfigMaps(operatorclient.OperatorNamespace).Get(ctx, "csr-signer-ca", metav1.GetOptions{})
 	if apierrors.IsNotFound(err) {
 		csrSignerCA = &corev1.ConfigMap{
-			ObjectMeta: metav1.ObjectMeta{Namespace: operatorclient.OperatorNamespace, Name: "csr-signer-ca"},
-			Data:       map[string]string{},
+			ObjectMeta: metav1.ObjectMeta{
+				Namespace: operatorclient.OperatorNamespace,
+				Name:      "csr-signer-ca",
+				Annotations: map[string]string{
+					annotations.OpenShiftComponent: "apiserver-auth",
```

Comment:
this looks solidly owned by kube-controller-manager.  The signer is being run in that process and this operator manages its lifecycle.